tests/cases/compiler/contextuallyTypedParametersPositionIncludesClassWithPrivateMember2.ts

```diff
@@ -0,0 +1,17 @@
+// @strict: true
+// @noEmit: true
+
+class Foo {
+  private foo = "foo";
+  doStuff(cb: (arg: string) => void) {}
+}
+
+interface FooLike {
+  doStuff: (cb: (arg: number) => void) => void;
+}
+
+declare function useIt(arg: Foo | FooLike): void;
+
+useIt({
+  doStuff: (arg) => {},
+});
```

Comment:
Do you mind adding the cases for just `Foo` and `FooLike`? like:

```ts
declare function useIt2(arg: Foo): void;

useIt2({
  doStuff: (arg) => {},
});

declare function useIt3(arg: FooLike): void;

useIt3({
  doStuff: (arg) => {},
});
```

Reply:
Sure thing, my dear friend - I just pushed them out :)